resources/servo.css

```diff
@@ -76,13 +76,21 @@ input[type="file"]::before {
   content: "Choose File";
 }
 
-input[type="file"] {
-  background: lightgrey;
+input[type="file"]::before {
   text-align: center;
   vertical-align: middle;
+  background: lightgrey;
+  border-top: solid 1px #EEEEEE;
+  border-left: solid 1px #CCCCCC;
+  border-right: solid 1px #999999;
+  border-bottom: solid 1px #999999;
```

Comment:
I wonder if we could update layout directly to show a button for file type instead of writing CSS here.